[2024-06-24T11:37:10.623+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-24T11:37:10.652+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.processing_tasks.inject_data_kyiv scheduled__2024-06-22T00:00:00+00:00 [queued]>
[2024-06-24T11:37:10.665+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.processing_tasks.inject_data_kyiv scheduled__2024-06-22T00:00:00+00:00 [queued]>
[2024-06-24T11:37:10.666+0000] {taskinstance.py:2306} INFO - Starting attempt 8 of 8
[2024-06-24T11:37:10.688+0000] {taskinstance.py:2330} INFO - Executing <Task(PostgresOperator): processing_tasks.inject_data_kyiv> on 2024-06-22 00:00:00+00:00
[2024-06-24T11:37:10.698+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1628) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-24T11:37:10.700+0000] {standard_task_runner.py:63} INFO - Started process 1634 to run task
[2024-06-24T11:37:10.699+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'processing_tasks.inject_data_kyiv', 'scheduled__2024-06-22T00:00:00+00:00', '--job-id', '157', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpn8qfy8px']
[2024-06-24T11:37:10.703+0000] {standard_task_runner.py:91} INFO - Job 157: Subtask processing_tasks.inject_data_kyiv
[2024-06-24T11:37:10.768+0000] {task_command.py:426} INFO - Running <TaskInstance: weather_dag.processing_tasks.inject_data_kyiv scheduled__2024-06-22T00:00:00+00:00 [running]> on host afc8096e8b86
[2024-06-24T11:37:10.965+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='processing_tasks.inject_data_kyiv' AIRFLOW_CTX_EXECUTION_DATE='2024-06-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='8' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-22T00:00:00+00:00'
[2024-06-24T11:37:10.968+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-24T11:37:10.968+0000] {sql.py:276} INFO - Executing: 
            INSERT INTO public.measures (timestamp, temp, humidity, clouds, wind_speed, city)
            VALUES (to_timestamp(1719014400), 
                   289.55,
                   78,
                   81,
                   1.56,
                   'Kyiv'
            );
[2024-06-24T11:37:10.980+0000] {base.py:84} INFO - Using connection ID 'ps_db_conn' for task execution.
[2024-06-24T11:37:10.994+0000] {base.py:84} INFO - Using connection ID 'ps_db_conn' for task execution.
[2024-06-24T11:37:11.000+0000] {sql.py:487} INFO - Running statement: 
            INSERT INTO public.measures (timestamp, temp, humidity, clouds, wind_speed, city)
            VALUES (to_timestamp(1719014400), 
                   289.55,
                   78,
                   81,
                   1.56,
                   'Kyiv'
            );, parameters: None
[2024-06-24T11:37:11.001+0000] {sql.py:496} INFO - Rows affected: 1
[2024-06-24T11:37:11.004+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-24T11:37:11.018+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=processing_tasks.inject_data_kyiv, run_id=scheduled__2024-06-22T00:00:00+00:00, execution_date=20240622T000000, start_date=20240624T113710, end_date=20240624T113711
[2024-06-24T11:37:11.076+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-24T11:37:11.090+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
